CMakeLists.txt

```diff
@@ -6,5 +6,12 @@ cmake_minimum_required(VERSION 3.5)
 # This example uses an extra component for common functions such as Wi-Fi and Ethernet connection.
 # set(EXTRA_COMPONENT_DIRS $ENV{IDF_PATH}/examples/common_components/protocol_examples_common)
 
+set(SDKCONFIG "${CMAKE_BINARY_DIR}/sdkconfig")
+set(SDKCONFIG_DEFAULTS "sdkconfig.defaults")
```

Comment:
Think this is redundant as sdkconfig.defaults is already the normal default config

Reply:
We need overrides such as CONFIG_ESPTOOLPY_FLASHSIZE="8MB", this is not the default flash size